CONTRIBUTING.md

```diff
@@ -253,9 +246,8 @@ e.g. `allCluster`, `allTyped`.
 Binary compatibility rules and guarantees are described in depth in the [Binary Compatibility Rules
 ](https://doc.akka.io/docs/akka/snapshot/common/binary-compatibility-rules.html) section of the documentation.
 
-Akka uses [MiMa](https://github.com/lightbend/mima) to
-validate the binary compatibility of incoming pull requests. If your PR fails due to binary compatibility issues, you may see
-an error like this:
+Pekko uses [MiMa](https://github.com/lightbend/mima) to validate the binary compatibility of incoming pull requests. If your 
+PR fails due to binary compatibility issues, you may see an error like this:
 
 ```
 [info] akka-stream: found 1 potential binary incompatibilities while checking against com.typesafe.akka:akka-stream_2.12:2.4.2  (filtered 222)
```

Comment:
we will not be doing mima checks until after we do a release - the mima checks prove that you have broken bin compatibility with a previous release - we have no releases to compare against. we can't compare to akka because we are changing package names, jar names and other things like that are not binary compatible

Reply:
That's a good point. @pjfanning : Should I then just keep it as-is with pekko:pekko-stream as eventually we will start doing this after the first release? Or should we remove for now and bring it back once a release is done. My vote is for the former, that is, keep it as-is with pekko:pekko-stream. Let me know.

Reply:
I have left it as is for now.